### PR TITLE
Commits 532-580

### PR DESCRIPTION
**Pull request by slmarii**

### CODE REVIEW

File: analysis/PX4-Autopilot.md [hunk 1]
```diff
@@ -8822,11 +8822,11 @@ MC_ROLL_P, MC_PITCH_P: increase maximum value to 12
 - use the same value for both
 - lower control latency allows increasing these gains
 ### Antipattern Category
-
+X
```
Comment (IvDinten) [inline analysis/PX4-Autopilot.md:8825]: Similar change as in commit #516, but a different conclusion? Could you clarify this decision?

File: analysis/PX4-Autopilot.md [hunk 2]
```diff
@@ -8886,11 +8886,11 @@ Before, the RC channel was published as fast as the fmu was running
...
 ### Keyword
 fast
 ### Note
-
+This commit does not change any performance-related features. timestam_last_signal wasn't updated and checkstyle correction.
 
 ## Commit #540
```
Comment (IvDinten) [inline analysis/PX4-Autopilot.md:8895]: This commit has been forgotten, @slmarii please analyse this commit.

File: analysis/PX4-Autopilot.md [hunk 3]
```diff
@@ -8918,11 +8918,11 @@ these values even further.
 
 increasing IMU_GYRO_CUTOFF allows for better tuning gains (increased P).
 ### Antipattern Category
-
+New:Hard-coded-fine-tuning
 ### Keyword
 increase
 ### Note
-
+Stack size increased from 1000 to 1100.
```
Comment (IvDinten) [inline analysis/PX4-Autopilot.md:8925]: @slmarii This note belongs to #540, so #541 was actually accidentally skipped, not #540.

File: analysis/PX4-Autopilot.md [hunk 4]
```diff
@@ -9019,10 +9019,11 @@ fast
 ### Message
 ManualSmoothingXY: velocity as criteria for direction change to prenvent fast acceleration at low spped
 ### Antipattern Category
-
+General:Hard-coding
 ### Keyword
 fast
 ### Note
+_hold_yaw is being used if finite, otherwise constant _yaw.
```
Comment (IvDinten) [inline analysis/PX4-Autopilot.md:9026]: This note (and thereby this analysis, including Antipattern category) belongs to commit #547, not this one.
This commit has been accidentally skipped. @slmarii please take a look and move this analysis to #547 and analyse #548.

File: analysis/PX4-Autopilot.md [hunk 5]
```diff
@@ -9033,11 +9034,11 @@ Improve the Crazyflie MAVLink tunnel to increase efficiency
 
 This change fragments MAVLink packets more efficiently and therefore increases the net throughput. This in turn makes the connection significantly more stable and the Crazyflie experience overall more usable.
 ### Antipattern Category
-
+New:Hard-coded-fine-tuning
 ### Keyword
 increase
 ### Note
-
+PWB Default Min increased from 0 to 20.
```
Comment (IvDinten) [inline analysis/PX4-Autopilot.md:9041]: Also the following change was made:
```
- int poll_ret = px4_poll(fds, 2, 1000);
+ int poll_ret = px4_poll(fds, 2, 500);
```

**Review by IvDinten — CHANGES_REQUESTED** [review]
Check comments for change requests.

File: analysis/PX4-Autopilot.md (hunk 5 above)
Comment (slmarii) [reply analysis/PX4-Autopilot.md:9041]: Will update the notes and Antipattern category sections.

File: analysis/PX4-Autopilot.md [hunk 6]
```diff
@@ -9361,11 +9362,11 @@ frsky_telemetry: increase stack size by 52 bytes
 
 As indicated in previous logs (268 bytes left).
 ### Antipattern Category
-
+New:Hard-coded-timing, New:Hard-coded-fine-tuning
```
[the remark this answers is not in the document]
Comment (slmarii) [reply analysis/PX4-Autopilot.md:9365]: Will update the commit

File: analysis/PX4-Autopilot.md (hunk 4 above)
Comment (slmarii) [reply analysis/PX4-Autopilot.md:9026]: Will correct these mistakes.

File: analysis/PX4-Autopilot.md (hunk 3 above)
Comment (slmarii) [reply analysis/PX4-Autopilot.md:8925]: Will update the commits

File: analysis/PX4-Autopilot.md (hunk 1 above)
Comment (slmarii) [reply analysis/PX4-Autopilot.md:8825]: My bad, I got confused about the meaning of pitching for an aircraft and control of roll axis. I will correct this commit.

File: analysis/PX4-Autopilot.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (IvDinten) [reply analysis/PX4-Autopilot.md:9367]: @slmarii It seem that the note for commit #569 was not updated with this change.

**Review by IvDinten — CHANGES_REQUESTED** [review]
Check comment(s) for change requests.

File: analysis/PX4-Autopilot.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (slmarii) [reply analysis/PX4-Autopilot.md]: My bad, I pushed another commit now.

**Review by IvDinten — APPROVED** [review]

**Pull request merged**